testbed_example/cli_interface.py

```diff
@@ -0,0 +1,53 @@
+'''
+The control software for an example testbed.
+
+Usage:
+  catkit start server [--port PORT_ID] [--simulated] [--config_path PATH]...
+  catkit start gui [--port PORT_ID]
+  catkit (-h | --help)
+  catkit --version
+
+Options:
+  -p, --port PORT_ID         The port number on which the testbed server operates.
```

Comment:
The indentation is a little bit messed up here, suggestion:
```suggestion
  -p, --port PORT_ID       The port number on which the testbed server operates.
```